src/module/data/actor-character.mjs

```diff
@@ -45,12 +56,37 @@ export default class GrimwildCharacter extends GrimwildActorBase {
 						max: 4,
 						initial: 1,
 						min: 0
-					})
+					}),
+					marked: new fields.BooleanField()
 				});
 				return obj;
 			}, {})
 		);
 
+		schema.thorns = new fields.NumberField({
+			integer: true,
+			min: 0
+		});
+
+		schema.features = new fields.StringField();
+		schema.conditions = new fields.StringField();
+		schema.backgrounds = new fields.SchemaField({
+			name: new fields.StringField(),
+			wises: new fields.ArrayField(new fields.StringField())
+		});
+		schema.traits = new fields.SchemaField({
+			are: new fields.ArrayField(new fields.StringField()),
+			not: new fields.StringField()
+		});
+		schema.desires = new fields.SchemaField({
+			are: new fields.ArrayField(new fields.StringField()),
+			not: new fields.StringField()
+		});
+		schema.bonds = new fields.SchemaField({
```

Comment:
I would make this an array of schema fields since multiple bonds will exist on each character.